tests/Controller/AjaxTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class AjaxTest extends WebTestCase
+{
+    public function testHomepage()
+    {
+        $client = static::createClient();
+
+        $client->request('GET', '/');
```

Comment:
Is this line of code touching the network/internet? If so, I'd recommend mocking the call/response so that this unit test can run when there is no internet connectivity.

Reply:
There is no internet connectivity involved :)